contrib/win32/win32compat/misc.c

```diff
@@ -167,7 +177,8 @@ nanosleep(const struct timespec *req, struct timespec *rem)
 		return -1;
 	}
 
-	li.QuadPart = -req->tv_nsec;
+	/* convert timespec to 100ns intervals */
+	li.QuadPart = -(timespec_to_nsec(req) / 100);
```

Comment:
What about tv_sec part of timespec?

Reply:
Whole the time structure "timespec" converts to nanoseconds (including tv_sec) in the function "timespec_to_nsec"